### ISSUE TITLE
gestire @result['artists'] che non sia un hash

### ISSUE THREAD

**Issue by gmgp** [slyver/spotifi_search#3]
se artists non è valorizzato con un hash come lo gestisci?

**Comment by slyver** [comment]
vero, non ho tenuto conto di un fallimento di JSON.parse, aggiungo un controllo che nel caso lo inizializzo con un hash con il campo error
